dev/unix/install.sh.in

```diff
@@ -217,11 +251,11 @@ notion_install() {
     if ! command grep -qc 'NOTION_HOME' "$NOTION_PROFILE"; then
       command printf "${PATH_STR}" >> "$NOTION_PROFILE"
     else
-      notion_warning "Your profile (${NOTION_PROFILE}) already mentions notion and has not been changed."
+      notion_warning "Your profile (${NOTION_PROFILE}) already mentions `notion` and has not been changed."
```

Comment:
This one should be "Notion" since it's talking about the product. Oh, the pedantry…